app/models/voxelytics/VoxelyticsDAO.scala

```diff
@@ -4,8 +4,9 @@ import com.scalableminds.util.time.Instant
 import com.scalableminds.util.tools.Fox
 import models.user.User
 import play.api.libs.json._
-import slick.jdbc.PostgresProfile.api._
-import utils.{ObjectId, SQLClient, SimpleSQLDAO}
+import utils.sql.SqlInterpolation.sqlInterpolation
```

Comment:
Any idea how to make this more concise? I'm afraid my "implicit"-foo is not sufficient to optimize this further.

Reply:
I added a commit, forwarding the implicit definition to the SimpleSQLDAO (from which the VoxelyticsDAO inherits, so no need for the import there). Is this what you had in mind?

Reply:
Yes! Perfect!